source/localizable/smart_cart/_order_object.html.md.erb

```diff
@@ -21,6 +21,7 @@ Name         | Type   | Value | Description
 `accept_options` | Object | | [Accept options](#accept-options) (available for orders with state `"open"`)
 `reject_options` | Object | | [Reject options](#reject-options) (available for orders with state `"open"`)
 `fulfilled_by_skroutz` | Boolean | | Whether the order is fulfilled by skroutz
+`fbs_delivery_note` | String | | Order's delivery note from skroutz warehouse
```

Comment:
`for fulfilled by Skroutz orders`

Also maybe Skroutz with capital S

Reply:
`for "fulfilled by Skroutz" orders` 
or `orders that are fulfilled by Skroutz`

Reply:
yes you're right 
768a8b12229c36a23a158649c94470b0928ceea1
897abc1fbb2af64db994bdfb87453682aeb0ff0e

Reply:
Updated 
f365907cfd4b358ffd95746fcaac23e75d54c356